extensions/kafka_producer.go

```diff
@@ -130,6 +130,12 @@ func (c *KafkaProducer) SendAPNSPush(topic, deviceToken string, payload, message
 		templateName,
 	)
 
+	if val, ok := pushMetadata["dryRun"]; ok {
+		if dryRun, _ := val.(bool); dryRun {
+			msg.DeviceToken = GenerateID(64)
```

Comment:
We could prefix our fake tokens with `FAKE` so if we look at the sent messages and/or feedbacks we can quickly know it was a dry run.

We could either do it here
```
msg.DeviceToken = fmt.Sprintf("FAKE-%s", GenerateID(64))
```

or change the `GenerateID` method.

Reply:
It's a really good idea. I'll change this.
Thanks for the suggestion.